### DISCUSSION TITLE
Bruno - Testimonials ❤️

### DISCUSSION THREAD

**Discussion by helloanoop** [usebruno/bruno#343] [Show and tell]
We are so happy to see the community love and reaction that Bruno has received over the past few days.
Our Github Stars⭐ are currently experiencing a vertical take-off. It's surreal.

People who use Bruno love the fully-offline, git-friendly, no-cloud-sync aspect of Bruno.

This discussion thread is our **Thank You Wall**.
Share your testimonials with us in the comments below and let others know how Bruno has helped you at work and your teams

Thank you for being a part of our community! ❤️
![image](https://github.com/usebruno/bruno/assets/9350733/cc1ae7b1-d0c5-4f13-9e6d-c4f5bee77ecf)

**Comment by l0new4nder3r** [comment]
Hi,

I got interested in the project a few months ago, when I was looking for a viable Postman alternative (for both personal and professional use), because (whatever they say) sending sometimes sensitive/private data to another American company was a no-go. 
Being able to keep a few curls "locally" should be an easy thing to do.

I seriously began playing with it a couple of weeks ago - when there were less than 500⭐ on the project.
I use it daily now, I am quite satisfied by how it is at the moment - _for my rather "simple needs"_.

**And this by itself is already amazing, so thank you @helloanoop, and thanks to you the other contributors!**

I really like the app, I like the way you manage the project.
I might (if I manage to find some more time and energy) try to contribute one day - in the code, I mean, even though I never played with node and React.
Until now I tried to regularly have a look at the issues, answering here and there to (hopefully) provide some assistance.

Seeing the issues (bugs or features) piling up faster and faster each day (even when some are resolved daily!) makes me wonder: it could be stressful for you to see them all coming. 
It could be overwhelming to want to address all that, answer and please everyone, on top of the coordination of the other contributors, validation of the PRs, all ASAP to keep with the momentum, etc.

My point is: do not rush it. 
If that happens, take some time for yourself, spend some quality time with the real-life Bruno 🐶.
Do not burn yourself out, all this can wait a few days!

Anyways, keep up the good work, and long live this project!

Cheers from France

**Comment by ptrdlbrg** [comment]
Thank you for your gift to the world. I sincerely appreciate your work on this project.

I found out about Bruno a couple days ago when people on Hacker News suggested switching away from Insomnia to Bruno. I was using Postman but I wasn't happy with it's proprietary data format and being closed source.  It's great that Bruno is both open source and uses a simple data format that can be read, version controlled, easily copied, etc. 

I've installed Bruno today. I got up and running on it quickly and have already used it a fair amount. I'm dropping Postman for Bruno.

Your efforts have brightened my day and improved my workflow. Thanks!

**Comment by RamiAwar** [comment]
I cannot count the number of times I've went on a rant about Postman and Insomnia and wished for an alternative that allowed for simple collaboration through version control.

In my opinion, Bruno makes for the ultimate API testing tool. Unlimited collaboration, easy importing of collections, and it's open source. I'm speechless. It literally took less than 5 minutes to convince my whole team to switch to Bruno from Postman. I've also started using it for dataline.app!

Thank you for taking on the responsibility of fixing API testing and building this!

**Comment by helloanoop** [comment]
There are no borders in opensource ❤️

![image](https://github.com/usebruno/bruno/assets/9350733/431c494a-cf10-4327-808c-bf67620992d9)

**Comment by mark05e** [comment]
Got a shout by @t3dotgg on https://www.youtube.com/watch?v=eqf9kD2XPn8

[<img src="https://i.ytimg.com/vi/eqf9kD2XPn8/maxresdefault.jpg" width="50%">](https://www.youtube.com/watch?v=eqf9kD2XPn8)

**Comment by marksimpson82** [comment]
I'm really enjoying using Bruno so far! VCS first, no middleman sucking up my data. Thank you for your work :-)

**Comment by langfeld** [comment]
After Insomnia turned to the dark side, I urgently needed a good API test program and found it in Bruno.
Simple and yet powerful (thanks to the powerful scripting function). 

Many thanks for this great program. If the Gold version becomes a one-time payment, I will buy it (even if I don't need the extra features).

Extra credits for the name and many greetings to the name giver 🦮

**Comment by RaphC** [comment]
Thank you for your work @helloanoop and all the bruno contributors. It's the best postman/insomnia alternative I've found : 
Easy to install, easy to use, extensible, sharing a bruno project through a VCS solution is really nice ... so many helpful  stuff but only what we really need !
Although its young age, this project is definitely the api testing tool you should use 

Thans again

**Comment by movoid12** [comment]
## Thanks to all devs behind bruno! Really great work and clean UI!

**Comment by myalban** [comment]
The most useful application of the year, and Bruno the dog seems great too!

**Comment by Napam** [comment]
Bruno has been a godsend for me personally and professionally. ⭐ I have completely replaced Postman/Insomnia for my team at work with Bruno!

- The UI is nice and simple, without bloat. 💯 
- The idea of storing the collections in a human friendly file structure which you can store wherever you want is so much better than the alternatives that obnoxiously push you to use their so called "solutions" (which are also paid). 🤮

Before Bruno I found myself fighting with Postman / Insomnia all the time, as I found their GUIs to be very clunky and bloated, and the collaboration features were just unneccesarily difficult to deal with. I dreaded everytime I had to maintain or setup Postman / Insomnia with team collaboration in mind. **Switching to Bruno really felt like removing weight from my shoulders.**

**Comment by bturner1273** [comment]
my insomnia config stopped working the other week and I decided I am not reconfiguring or switching back to postman.

The lack of OAuth2 support means you will have to log into your IdP with a custom pre-request script and append bearer tokens yourself, but I don't see this as a major issue. The scripting and support for environment variables (managed by you in .env OR by bruno which excludes 'secret' environment variables from output files) made any custom behavior a breeze.

I would certainly recommend this. The setup took about an hour even with some custom scripting and I am sure everyone will rejoice in uninstalling insomnia and getting up and running with bruno after a git pull and 5 minutes to setup the .env file.

**Comment by soura-b** [comment]
I am an API newbie, and for a while I was worried about how easy it was to use Bruno &mdash; which meant _of course_ I was missing something &mdash; till I realized that this is exactly why it exists. Thank you for creating this.

**Comment by truongtrnghia** [comment]
Thanks a lot for keeping things offline. I switched from Postman to Insomnia and now Bruno only for that reason

**Comment by vladzima** [comment]
I think more developers should know about Bruno as soon as possible, this should be in top-5 dev tools for everyone ever touching an API. The minute I opened the docs I got smashed by all the various options that can be implemented providing unlimited extensibility of the app, CLI and language. _Not just your another curl wrapper._

**Comment by glen-polo** [comment]
I've been using Postman for years, and while it's wonderful, it was missing the exact things that Bruno is solving...for FREE.  I've transitioned my team to use Bruno now and spreading the news to other teams.  Overall it's very clean, simple, and the response time for bugs is greatly appreciated as well!  It's the small things that make the biggest differences imo, and I hope Bruno continues to excel and grow.

**Comment by qiutq1** [comment]
I have cleared all the hurdles that prevented me from switching to Bruno, get Postman out for its bad decisions!

**Comment by Vikas1712** [comment]
Switching from Postman to Bruno API was a game-changer for our team. The Transition was seamless and we were up and running with Bruno in no time. It's not just about tools, though, the support we've received has been top-notch. @helloanoop in particaular deserves a special mention for his prompt responses.

**Reply by melroy89** [reply]
Agreed.. I never used Postman.. However in the recent past I was looking into Postman.. I tried it.. And I didn't like it at all.. Then I tried Insomnia.. Also didn't like it.. It's all so slow.. And finally I found Bruno!

I'm Bruno fan since day 1 and have a golden edition license. I never looking back to those "alternative" anymore.

**Comment by HSGamer** [comment]
Love the Git-friendly part. I had difficulty sharing the API tests with my team until I found this.

**Comment by Skeeve** [comment]
I first used postman, which failed me in regards to SOCKS proxy. Then insomnis, which got unusable.

Now I'm here and after understanding how bruno works, I really like it. Will try and convince my boss to pay a license for my colleague and me.

**Reply by melroy89** [reply]
I'm using Bruno for at least 1 year using the Golden license (maybe for a company you should go with Ultimate Edition), anyhow I will never go back to either Insomnia or Postman. Bruno is so much faster.

**Reply by Skeeve** [reply]
> maybe for a company you should go with Ultimate Edition

I do not see any advantage in that. The additional features are nothing we really need.

**Reply by melroy89** [reply]
You will help the project to be sustainable

**Reply by Skeeve** [reply]
> You will help the project to be sustainable

Sure. Please tell my boss.

**Reply by melroy89** [reply]
Sure. What is his mail?

**Reply by Skeeve** [reply]
> Sure. What is his mail?

🤣

**Comment by devdilson** [comment]
I always had issues with Postman and there messy collection files. With Bruno, I can easily version control requests and quick share with our team. 
Really nice tool!

**Comment by Felix00643298** [comment]
Thank you.

**Comment by prasoonpankaj** [comment]
Bruno made it on Thoughtworks Technology Radar - [https://www.thoughtworks.com/radar/tools/bruno](https://www.thoughtworks.com/radar/tools/bruno)! It's a big deal!

**Comment by fantpmas** [comment]
Wrote something for our company blog: https://www.inthepocket.com/blog/bruno-the-lean-and-git-friendly-api-client-weve-been-waiting-for 
Thanks for making Bruno folks, really appreciated!

**Comment by nathaniel-d-ef** [comment]
I recorded a quick demo showing off the gRPC functionality of Bruno against an Arrow Flight server. I'm so happy to finally be rid of Postman, great work all! [https://youtu.be/h9ZT6-YWw2U?si=q21WoqYRO67bQXHh](https://youtu.be/h9ZT6-YWw2U?si=q21WoqYRO67bQXHh)

**Reply by sanish-bruno** [reply]
@nathaniel-d-ef Hey, thanks for the video, we would love to hear your suggestions on improving the gRPC workflows, please do add your suggestions to the following discussion https://github.com/usebruno/bruno/discussions/5447 . Your suggestion will help us improve the experience!

**Comment by dbPietah** [comment]
What an absolute breath of fresh air after trying to work with Postman and Insomnias bloated AI & cloud first shenanigans. I introduced it to other developers and analysts at work and they where giddy to adopt it.

Love from Belgium

**Reply by helloanoop** [reply]
Thank you ❤️
